src/bos/operators/base.py

```diff
@@ -68,7 +69,7 @@ def __init__(self):
         #self.cfs = CFSClient()
         #self.hsm = HSMClient()
         #self.ims = IMSClient()
-        #self.pcs = PCSClient()
+        self.pcs = PCSClient()
```

Comment:
Why instantiate all of the clients in the base operator rather than in only the clients that need them?

Reply:
For simplicity, more than any other reason. I would not be opposed to doing it differently.

Reply:
It's a potential future enhancement. I'm not so opposed to doing it this way that it would need to be changed now. I just wanted to make sure I wasn't missing a trick where the base operator was closing everything on exit and doing it in the other operators would complicate things.